src/_layouts/default.html

```diff
@@ -4,7 +4,7 @@
   {%- include head.html -%}
 
   <body class="body body--{{ page.layout }}">
-
+    <div class="body-blackout"></div>
```

Comment:
I think this class should be something like "modal-background" to make it more obvious why it's here.

Reply:
Also, I really wish that clicking on this modal background would also dismiss the modal. It's annoying to snipe that "X" close button.